docs/content/latest/architecture/docdb-sharding/tablet-splitting.md

```diff
@@ -195,21 +206,94 @@ Tablet UUID                       Range                                 Leader
 a89ecb84ad1b488b893b6e7762a6ca2a  key_start: "\177\377" key_end: ""     127.0.0.3:9100
 ```
 
-{{< note title="Note" >}}
+{{< note title="Important" >}}
 
-There are a few important things to note here. 
-* The original tablet `9991368c4b85456988303cd65a3c6503` is no longer present in the system. It has been replaced with two new tablets. 
-* The tablet leaders are now spread across two nodes, in order to evenly balance the tablets for a table across the nodes in the cluster.
+* The original tablet `9991368c4b85456988303cd65a3c6503` no longer exists and has been replaced with two new tablets.
+* The tablet leaders are now spread across two nodes in order to evenly balance the tablets for the table across the nodes of the cluster.
 
 {{</note >}}
 
-## Automatic splitting
+## Automatic tablet splitting [BETA]
 
 {{< note title="Note" >}}
 
-This feature is currently in [BETA](../../../../faq/general/#what-is-the-definition-of-the-beta-feature-tag).
+Automatic tablet splitting is currently in [BETA](../../../faq/general/#what-is-the-definition-of-the-beta-feature-tag).
 
 {{< /note >}}
 
-Design for automatic splitting can be reviewed on [GitHub](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-automatic-tablet-splitting.md). While the broader feature is [work-in-progress](https://github.com/yugabyte/yugabyte-db/issues/1004), tablets can be automatically split for a few scenarions starting the v2.2 release. Detailed documentation is coming soon.
+Automatic tablet splitting enables resharding of data in a cluster automatically while online, and transparently to users, when a specified size threshold has been reached.
+
+For details on the architecture design, see [Automatic Re-sharding of Data with Tablet Splitting](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/docdb-automatic-tablet-splitting.md). While the broader feature is [work-in-progress](https://github.com/yugabyte/yugabyte-db/issues/1004), tablets can be automatically split for a few scenarios starting in the v2.2 release.
```

Comment:
I would remove the last sentence. Better to just call out that the feature is in BETA, than say that `the broader feature is a work in progress`, since that sounds more like something that users would not be incentivized to even try...